File: PracticasParciales/Final/ejercicio2/cessa/app/Models/Factura.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Factura extends Model
{
    /** @use HasFactory<\Database\Factories\FacturaFactory> */
    use HasFactory;
    protected $fillable = [
        'empresa',
        'nro_factura',
        'ci',
        'nombre_completo',
        'periodo',
        'monto',
        'estado',
    ];
}

File: PracticasParciales/Final/ejercicio2/notificadorPago/notificadorPago/Factura.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace notificadorPago
{
    public class Factura
    {
        public string Empresa { get; set; }
        public string NroFactura { get; set; }
        public string CI { get; set; }
        public string NombreCompleto { get; set; }
        public string Periodo { get; set; }
        public decimal Monto { get; set; }
        public string Estado { get; set; }
    }
}